setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='django-markwhat',
-    version=".".join(map(str, __import__('django_markwhat').__version__)),
```

Comment:
Okay, cool.. 

Yeah, the patch is ready to merge as far as I'm concerned. Thanks